kafkaesk/app.py

```diff
@@ -317,14 +327,16 @@ async def raw_publish(
         stream_id: str,
         data: bytes,
         key: Optional[bytes] = None,
-        headers: Optional[Dict[str, bytes]] = None,
+        headers: Optional[List[Tuple[str, bytes]]] = None,
     ) -> Awaitable[aiokafka.structs.ConsumerRecord]:
         logger.debug(f"Sending kafka msg: {stream_id}")
         producer = await self._get_producer()
         tracer = opentracing.tracer
 
         if headers is None:
-            headers = {}
+            headers_unique = {}
+        else:
+            headers_unique = {k: v for k, v in headers}
```

Comment:
@mdscruggs-onna i have changed a bit the headers logic, we don't really want to squash duplicated keys, since in some use cases that can be used as a way to send lists of headers.